src/main/webapp/player/simid_player.html

```diff
@@ -46,6 +46,10 @@
     window.switchCreative = (value) => {
       document.getElementById('creative_url').value = '../creatives/' + value;
       // populate ad parameters with some hard coded values
+      document.getElementById('geolocation').style.visibility = 'hidden';
+      if (value == 'maps/simid_map.html') {
```

Comment:
Keep this as part of the switch, no need to have an extra if statement if you're gonna use a switch anyway.

Reply:
Fixed